asm/include/overlay_01_021F1AFC.inc

```diff
@@ -100,12 +100,12 @@
 .public ov01_PlayerAvatar_OrrTransitionFlags
 .public ov01_PlayerAvatar_ApplyTransitionFlags
 .public ov01_021F1D94
-.public ScriptCallTask_Surf
+.public CallFieldTask_Surf
```

Comment:
CallTask_FieldSurf? Or is there a reason to separate CallTask_ with CallFieldTask?